pootle_pytest/fixtures/site.py

```diff
@@ -113,3 +113,22 @@ def site_matrix(site_root):
             # add a TP to the project for each language
             tp = TranslationProjectFactory(project=project, language=language)
             _add_stores(tp)
+
+
+@pytest.fixture
+def site_matrix_with_terminology(site_matrix):
+    from pootle_project.models import Project
+    from pootle_language.models import Language
+    from pootle_pytest.factories import (
+        ProjectFactory, TranslationProjectFactory
+    )
+
+    source_language = Language.objects.first()
+    terminology_project = ProjectFactory(code='terminology',
+                                         source_language=source_language,
+                                         check_style='terminology')
+    for language in Language.objects.all():
+        # add a TP to the terminology project for each language
+        tp = TranslationProjectFactory(project=terminology_project,
+                                       language=language)
+        _add_stores(tp, (1,1))
```

Comment:
...although im still struggling to understand why this isnt being picked up by pep8